dotnet/src/SemanticKernel.Abstractions/Orchestration/IFunctionRunner.cs

```diff
@@ -15,11 +16,13 @@ public interface IFunctionRunner
     /// </summary>
     /// <param name="skFunction">Target function to run</param>
     /// <param name="variables">Input to process</param>
+    /// <param name="requestSettings">Request settings to override</param>
     /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
     /// <returns>Result of the function composition</returns>
     Task<FunctionResult> RunAsync(
         ISKFunction skFunction,
         ContextVariables? variables = null,
+        AIRequestSettings? requestSettings = null,
         CancellationToken cancellationToken = default);
```

Comment:
Not expecting anyone will have implemented this but if they have this makes the change a breaking one, so we will need to add that tag.

Reply:
You are right, added an FunctionRunner extensions class to handle that.